app/frontend/src/views/admin/InspectionDashboard.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-container class="dashboard text-center">
+    <BaseSecure inspector>
+      <div v-if="inspectionDashboardUrl" class="dashboard-container">
+        <iframe :src="inspectionDashboardUrl" />
+      </div>
+      <div v-else>
+        <p>No dashboard has been configured.</p>
+        <router-link :to="{ name: 'Admin' }">
+          <v-btn color="primary" class="about-btn" large>
+            <v-icon left>home</v-icon>
+            <span>Submissions</span>
+          </v-btn>
+        </router-link>
+      </div>
+    </BaseSecure>
```

Comment:
Seems to be a copy-paste of Dashboard.vue with some renames?
So I think we should consider componentizing this block with the script and style.

Reply:
I agree with that notion as well. It'll give us a better dashboard component to leverage in the long run.

Reply:
sure. do you think we want to use slots and make it a NotFound.vue base component ?

Reply:
No, a dashboard component that takes in params about what dashboard to load.

Reply:
of course.. good idea...done